src/Core/DevOps/StaticAnalyze/PHPStan/common.neon

```diff
@@ -12,6 +12,10 @@ parameters:
     checkFunctionNameCase: true
     checkInternalClassCaseSensitivity: true
 
+    # Would otherwise flag usage of internal classes from different namespaces
```

Comment:
gives ~2k errors :see_no_evil: we might want to consider it at a later point nevertheless. 
same as in our case, internal classes and method from 3rd party packages might change at any point and could cause breaks